contrib/net/http/http.go

```diff
@@ -38,3 +38,10 @@ func (mux *ServeMux) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// we need to wrap the ServeHTTP method to be able to trace it
 	internal.Trace(mux.ServeMux, w, r, mux.service, resource, mux.Tracer)
 }
+
+// WrapHandler wraps an Handler with a tracer.
```

Comment:
I would prefer this to go into a bit more detail:
```go
// WrapHandler wraps an http.Handler with the given tracer using the
// specified service and resource.
```

Reply:
done